welcome-message/main.scss

```diff
@@ -1,15 +1,16 @@
 @import 'o-visual-effects/main';
+@import 'o-icons/main';
 
 @if nUiApply('welcomeMessage') {
 
 	@include nUiMaybeCriticalStart('welcomeMessage');
 
-	.n-banner-message,
+	.n-welcome-banner,
 	.n-welcome-message {
 		position: relative;
 		padding: 16px 12px;
 		// not in o-colors ಠ_ಠ
-		background: #FADBCB;
+		background: #fadbcb;
```

Comment:
This is the old banner background, nearest valid colour is washed out on our background.
![screen shot 2016-09-19 at 12 16 43](https://cloud.githubusercontent.com/assets/996106/18630182/a66eff90-7e62-11e6-9085-3596ea733d93.png)